.github/workflows/run-tests.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     - name: Lint config.py with flake8
       run: make flake-config
     - name: Install codespell
-      run: pip install codespell
+      run: pip install "codespell @ https://github.com/Freed-Wu/codespell/archive/refs/heads/iss2055.zip" tomli  # fork that supports pyproject.toml
```

Comment:
-1 on this, let's push for https://github.com/codespell-project/codespell/issues/2055 instead

Reply:
... I see why you did this, so you could get rid of `setup.cfg` entirely. Let's keep it then and I'll fix this once the upstream PR is merged